app/src/main/java/org/eyeseetea/malariacare/layout/utils/LayoutUtils.java

```diff
@@ -219,4 +261,89 @@ public static String getAppNameColorString() {
                 R.color.appNameColor);
         return String.format("%X", appNameColor).substring(2);
     }
+
+    public static void updateSurveyActionBarChartAddingQuestion(ActionBar actionBar, long surveyId, final boolean compulsory){
+    }
+
+    public static void updateSurveyActionBarChartRemovingQuestion(ActionBar actionBar, long surveyId, final boolean compulsory){
+
+    }
+
+    private static void updateChart(SurveyAnsweredRatio surveyAnsweredRatio,
+            DoublePieChart doublePieChart) {
+        doublePieChart.createDoublePie(surveyAnsweredRatio.getMandatoryStatus(),
+                surveyAnsweredRatio.getTotalStatus());
+    }
+
+    public static void updateSurveyActionBarChart(ActionBar actionBar, long surveyId){
+
+        final DoublePieChart doublePieChart =
+                (DoublePieChart) actionBar.getCustomView().findViewById(R.id.action_bar_chart);
+        doublePieChart.setVisibility(View.VISIBLE);
+        ISurveyAnsweredRatioRepository surveyAnsweredRatioRepository =
+                new SurveyAnsweredRatioRepository();
+        SaveSurveyAnsweredRatioUseCase saveSurveyAnsweredRatioUseCase =
+                new SaveSurveyAnsweredRatioUseCase(surveyAnsweredRatioRepository);
+        saveSurveyAnsweredRatioUseCase.execute(surveyId,
```

Comment:
saveSurveyAnsweredRatioUseCase should not be executed from layoutUtils because is not layout or utils